taskvine/src/manager/taskvine.h

```diff
@@ -44,6 +44,7 @@ typedef enum {
 	VINE_NOCACHE  = 0, /**< Do not cache file at execution site. (default) */
 	VINE_CACHE    = 1, /**< Cache file at execution site for later use. */
 	VINE_UNPACK   = 2, /**< Unpack this archive (.tar .tgz .zip) into a directory on arrival. */
+	VINE_PONCHO_UNPACK, /**< Unpacks a Poncho environment into a directory on arrival. */
```

Comment:
Note that this enumeration is a bitfield where multiple flags can be set at once,
so each line needs a distinct value which is a power of two.